<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PC-DEV15" failures="1" tests="8" name="com.saphir.platforme.ActionSimplifierRunner" time="788.939" errors="0" timestamp="2023-10-13T01:29:32 CEST" skipped="0">
  <testcase classname="com.saphir.platforme.ActionSimplifierRunner" name="runScenario" time="82.143"/>
  <system-out/>
  <testcase classname="com.saphir.platforme.ActionSimplifierRunner" name="runScenario" time="139.710"/>
  <system-out/>
  <testcase classname="com.saphir.platforme.ActionSimplifierRunner" name="runScenario" time="68.221"/>
  <system-out/>
  <testcase classname="com.saphir.platforme.ActionSimplifierRunner" name="runScenario" time="111.182"/>
  <system-out/>
  <testcase classname="com.saphir.platforme.ActionSimplifierRunner" name="runScenario" time="128.869"/>
  <system-out/>
  <testcase classname="com.saphir.platforme.ActionSimplifierRunner" name="runScenario" time="67.850">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.saphir.platforme.moduleAction.stepdefs.FicheActionStepDefinition.Vérifier_les_donnes_de_traçabilite(FicheActionStepDefinition.java:1469)
at ✽.Vérifier les donnes de traçabilite(file:///E:/qualipro/trunk/platforme_test_automation_backend/src/main/java/com/saphir/platforme/features/moduleAction/ActionSimp.feature:154)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.saphir.platforme.ActionSimplifierRunner" name="runScenario" time="122.390"/>
  <system-out/>
  <testcase classname="com.saphir.platforme.ActionSimplifierRunner" name="runScenario" time="68.574"/>
  <system-out/>
</testsuite> <!-- com.saphir.platforme.ActionSimplifierRunner -->
